### DISCUSSION TITLE
Copy settings in Library module

### DISCUSSION THREAD

**Discussion by nadavrotem** [rsjaffe/MIDI2LR#558] [Q&A] [answered]
Hi,
I am trying to Copy settings while I am on the Library Module, There is nothing built in (or am I missing something?) so I assigned Keyboard shortcut 1 as Shift+Ctrl+C and it works though I was wondering how do I assign the "Enter" key... 
As it is now I will still have to hit the key on the computer keyboard and I am trying to avoid that...
I saw in an 2017 YouTube video that there was an option to Copy without pressing enter, is that still available?
Thank you

**Comment by rsjaffe** [comment, marked as answer]
Make a new keyboard shortcut, and instead of typing in a single key, type `return`. See https://github.com/rsjaffe/MIDI2LR/wiki/Plugin-Options-Dialog#keyboard-shortcuts for more information.

The old copy command was removed as Adobe kept on changing things without documenting the changes, so I couldn't keep it up-to-date.

**Reply by nadavrotem** [reply]
Thank you!
